core/trino-main/src/main/java/io/trino/execution/executor/TaskExecutor.java

```diff
@@ -187,20 +204,29 @@ public TaskExecutor(
             int guaranteedNumberOfDriversPerTask,
             int maximumNumberOfDriversPerTask,
             VersionEmbedder versionEmbedder,
+            Duration interruptRunawaySplitsTimeout,
```

Comment:
interrupStuckSplitsTimeout/interruptStuckSplitsDelay (should match config option)